### DISCUSSION TITLE
How to add a deployment badge for GH Pages?

### DISCUSSION THREAD

**Discussion by Buggem** [community/community#36110] [Other Feature Feedback, Questions, & Ideas] [answered]
How do show users the deployment status for GitHub Pages? I'd really like to include a badge indicating the deployment in my README.md, but don't know how. I've searched everywhere but there's no result.

**Comment by DSDmark** [comment, marked as answer]
It's simpler

### click on action 

- ![Screenshot from 2022-10-14 13-03-34](https://user-images.githubusercontent.com/85744816/195789107-68065aef-d9d7-4db6-a2ad-fea29224f1bd.png)

### then select your action 

- ![Screenshot from 2022-10-14 13-04-09](https://user-images.githubusercontent.com/85744816/195789219-551dafbf-f65b-4166-92b7-f9094a1ffd8b.png)

### and after on right side there is three dots appears 

- ![Screenshot from 2022-10-14 13-04-43](https://user-images.githubusercontent.com/85744816/195789379-acb82c0f-b14a-4ac3-83d6-d3bcd7b20bbb.png)

- ![Screenshot from 2022-10-14 13-05-47](https://user-images.githubusercontent.com/85744816/195789494-34f33217-e668-408a-a872-6916e8911385.png)

### click on it and copy and pest on README file

**Reply by Buggem** [reply]
Thanks! I got an answer within seconds. Great answer.
